src/java/com/vaklinov/zcashui/ZCashUI.java

```diff
@@ -570,6 +571,12 @@ public static void possiblyCreateHUSHConfigFile()
 			configOut.println("rpcpassword=Pass" + Math.abs(r.nextInt()) + "" + 
 			                                       Math.abs(r.nextInt()) + "" + 
 					                               Math.abs(r.nextInt()));
+			configOut.println("addnode=node.myhush.network");
```

Comment:
Please remove the same nodes here and update stilgar to stilgar.myhush.org. The host zdash.suprnova.cc seems to be good still, that can be kept. And dnsseed.myhush.org